engine/service.go

```diff
@@ -562,25 +564,57 @@ func (e *Engine) nuke(ctx context.Context, opts NukeOptions, op errors.Op) error
 
 // resolveServices resolves a list of services from either a list of service names or a playlist name.
 //
-// If both serivceNames and playlistName are provided, an error will be returned. Mixing service names
+// If both serviceNames and playlistName are provided, an error will be returned. Mixing service names
 // is not supported.
 //
 // If neither serviceNames nor playlistName are provided, then behaviour depends on the value of requireOne.
 // In this case, if requireOne is true, an error will be returned since at least one of serviceNames or playlistName
 // was required. Otherwise, both the returned slice and error will be nil, which can be treated as an empty slice
 // of services.
-func (e *Engine) resolveServices(op errors.Op, serviceNames []string, playlistName string, requireOne bool) ([]service.Service, error) {
-	if len(serviceNames) > 0 && playlistName != "" {
+func (e *Engine) resolveServices(op errors.Op, serviceNames []string, playlistName string, serviceTags map[string]string, requireOne bool) ([]service.Service, error) {
+	if len(serviceNames) > 0 && playlistName != "" && len(serviceTags) == 0 {
```

Comment:
Why would we restrict this validation to only the case where we aren't using service tags? even if we have multiple tags, we shouldn't be accepting both a playlist and a list of services

Reply:
you're right, that must have been a relic from when i was trying things out. i removed that extra check